spec/rails_helper.rb

```diff
@@ -0,0 +1,18 @@
+require "spec_helper"
+
+# Load the ManageIQ environment
+ENV['RAILS_ENV'] ||= 'test'
+require File.expand_path("../manageiq/config/environment", __FILE__)
+
+require "manageiq/graphql"
+require 'rspec/rails'
+
+# Dir[Rails.root.join("spec/shared/**/*.rb")].each { |f| require f }
+Dir[Rails.root.join("spec/factories/**/*.rb")].each { |f| require f }
+
+RSpec.configure do |config|
+  config.use_transactional_fixtures = true
+  config.infer_spec_type_from_file_location!
+  config.filter_rails_from_backtrace!
```

Comment:
Pedantic, but separate the following include with the metadata includes with a space here.